src/components/LocationPin.js

```diff
@@ -1,13 +1,24 @@
 import React from 'react';
 import './LocationPin.css';
+import { Icon } from '@iconify/react';
+import InfoWindow from '../components/InfoWindow.js';
+import location12Filled from '@iconify-icons/fluent/location-12-filled';
 
-export default function LocationPin({ img }) {
-  return (
-    <div className="pin">
-      <div
-        className="pin-fill"
-        style={{ backgroundImage: `url(${img})` }}
-      ></div>
-    </div>
+export default function LocationPin({ img, $dimensionKey }) {
+  return !img ? (
```

Comment:
Could you change the condition here from `!img` to `img` and swap the return values? That might read a little more easily.

Reply:
Looks like this still needs to be done?

Reply:
Just fixed this!